scripts/test

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
```

Comment:
One of the frustrations I have with our current test scripts are how they work with file watching.  When I'm writing code, I tend to have the tests running continuously in the container using watch mode.  I'd like to offer a simple way for teams to do the same as I'm sure I'm not alone in running tests in watch mode.

With our current setup, the only real way to do this is to change the command in the `docker-compose.test.yaml` file and remember to not commit that change.  That's a bit of friction I'm not a fan of, especially as I seem to keep committing that change causing the CI to get stuck watch the files!

What I would like us to do is add a default `docker-compose.test.watch.yaml` file so we don't need to keep amending the commands.

In test scripts I've written elsewhere I have adapted test scripts such as this to use this file depending on whether a `-w` argument is passed.

[Here](https://github.com/johnwatson484/dream-league-api/blob/master/scripts/test) is an example of a test script I've written almost exactly like this but also allows file watching.